defs.bzl

```diff
@@ -20,10 +21,21 @@ def maven_install(
         repositories = [],
         artifacts = [],
         fetch_sources = False):
+
+    repositories = parse.parse_repository_spec_list(repositories)
+    repositories_json_strings = []
+    for repository in repositories:
```

Comment:
nit: inline `parse.parse_repository_spec_list(repositories)` into the for-statement here, so we don't need to overwrite the variable or create a new one. same for artifacts.